test/WebSites/InProcessWebSite/Startup.cs

```diff
@@ -753,5 +753,11 @@ private void Shutdown(IApplicationBuilder app)
             }
             await context.Response.WriteAsync("Response End");
         }
+
+        private async Task GetIsConsoleRedirected(HttpContext context)
+        {
+            await context.Response.WriteAsync($"{Console.IsOutputRedirected}");
```

Comment:
I like the test, and I do think it makes more sense for `IsRedirected` to return false, since this should reflect the managed-code state of Output/Error, not what has been done by the process hosting the managed runtime.

Reply:
I need to investigate this test. It is still returning IsRedirected=true. I'll compare with file logging and nooping.

Reply:
So even without redirecting native stdout and stderr, managed believes that Console is always redirected. This may be some state that IISExpress/IIS set-up during process start to make managed believe that console already redirected.